WordPress/Classes/ViewRelated/Voice/VoiceToContentView.swift

```diff
@@ -128,26 +131,44 @@ private struct VoiceToContentRecordingView: View {
     @State private var isRecording = true
 
     var body: some View {
-        VStack(spacing: 34) {
-
+        VStack {
             Spacer()
-
             VStack(spacing: 8) {
-                // Fallback on earlier versions
                 if #available(iOS 17.0, *) {
                     Image(systemName: "waveform")
-                        .font(.system(size: 80, weight: .medium))
+                        .resizable()
+                        .scaledToFit()
+                        .frame(width: 80)
                         .foregroundStyle(Color(uiColor: .brand))
                         .symbolEffect(.variableColor, isActive: true)
+                } else {
+                    Image("audio-waveform")
```

Comment:
(nit) `Image(systemName: "waveform")` is available on iOS 15. The only part that is not is `symbolEffect`.

Reply:
Fixed! [bdd9077](https://github.com/wordpress-mobile/WordPress-iOS/pull/23258/commits/bdd9077951e9258626b59ae667f10fa549332a03)